src/routers/apps.py

```
@@ -1,12 +1,15 @@
-import traceback
+import logging
 
-from fastapi import APIRouter, Depends, HTTPException
+from fastapi import APIRouter, Depends
 from sqlalchemy.exc import IntegrityError
```

Ruff (0.8.2)

`sqlalchemy.exc.IntegrityError` imported but unused

Remove unused import: `sqlalchemy.exc.IntegrityError`

(F401)